### PR TITLE
Add links from Tutorial sections to relevant Samples code.

### PR DESCRIPTION
**Pull request by rfreytag**
Thought it helpful to link the Tutorial sections directly to the referenced Samples source code.

### CODE REVIEW

**Comment by rfreytag** [comment]
This is my first pull request on GitHub so am making it a 'draft pull request.'  Liking Elmish.WPF a lot; thanks.  Hope I can help the project.

**Comment by the-not-mad-psychologist** [comment]
> This is my first pull request on GitHub so am making it a 'draft pull request.'

Drafts are for while you're still working on PRs. When you're ready for someone to review the PR and consider merging it, mark it "Ready for Review".

File: TUTORIAL.md [hunk 1]
```diff
@@ -377,7 +377,7 @@ Additionally, there is a section explaining how most dispatching bindings allow
 
 ### One-way bindings
 
-*Relevant sample: SingleCounter*
+*Relevant sample: SingleCounter - ([C# wrapper](https://github.com/elmish/Elmish.WPF/tree/master/src/Samples/SingleCounter) which calls [F# core](https://github.com/elmish/Elmish.WPF/tree/master/src/Samples/SingleCounter.Core))*
```
Comment (TysonMN) [inline TUTORIAL.md:380]: Adjust your links so they have this format:

> XAML views and F# core

with the links on the obvious words.

**Review by TysonMN — CHANGES_REQUESTED** [review]
Thanks!

In addition to using relative links, just one more minor adjustment.

**Comment by rfreytag** [comment]
Seems my method of rewriting my repos branch history by deleting a remote branch closed this pull request.  I have issued another identical pull request with the changes specified by marner2 and TysonMN.  Thanks, Richard F.

**Pull request closed without merging**